### ISSUE TITLE
Cannot destructure property `Serializer` of `F6` as it is undefined.

### ISSUE THREAD

**Issue by chientrm** [fastify/fastify#4702]
### Prerequisites

- [X] I have written a descriptive issue title
- [x] I have searched existing issues to ensure the bug has not already been reported

### Fastify version

4.16.0

### Node.js version

18.15.0

### Operating system

Linux

### Operating system version (i.e. 20.04, 11.3, 10)

Ubuntu 22.10

### Description

<img width="660" alt="Screen Shot 2023-04-26 at 9 03 04 AM" src="https://user-images.githubusercontent.com/70304839/234447314-4bbdd8d1-6061-49e8-9f47-f534bdfcf5f8.png">

### Steps to Reproduce

`api.ts`
```ts
import fastify from "fastify";

fastify({ logger: true })
  .listen({ port });
```

`package.json`
```json
{
  "scripts": {
    "dev": "npm run build && node --enable-source-maps build_api/api.js | pino-pretty",
    "build": "esbuild api/api.ts --sourcemap --outfile=build_api/api.js --bundle --minify --platform=node"
  },
  "devDependencies": {
    "esbuild": "^0.17.18",
    "fastify": "^4.16.0"
  }
}
```

**Comment by climba03003** [comment]
We need to update the version inside [`@fastify/fast-json-stringify-compiler`](https://github.com/fastify/fast-json-stringify-compiler/blob/main/package.json#L38) and release.
Then update the version inside [`fastify`](https://github.com/fastify/fastify/blob/main/package.json#L184)
So, it force `npm` to reinstall the latest version of `fast-json-stringify`.

Anyway, as an alternative approach.
You can run `npm update fast-json-stringify` to reinstall the latest version.

**Comment by patoi** [comment]
I think the https://github.com/fastify/fastify/issues/4703 is related to this.

**Comment by mcollina** [comment]
Fixed in main, releasing v4.16.1 asap.

(Updating your dependency tree would have fixed it as well).

**Comment by Eomm** [comment]
Follow up - https://github.com/fastify/fast-json-stringify/issues/624
